source/_patterns/01-molecules/51-borrower-dashboard/01-whats-next-content/18-auto-repay.mustache

```diff
@@ -0,0 +1,8 @@
+<p>
+	<strong>Sign up to repay automatically</strong>
+</p>
+<p>Never miss a payment by signing up to automatically make payments through your PayPal account. As your Kiva payments impact your business credit score, we want to help you make every payment on time. Click the button below to learn more.</p>
+
+<div class="text-center">
+	<a class="whats-next-action" href="https://kiva.org/my/borrower/auto-repayments">Learn more</a>
```

Comment:
but its also not a big deal, and it does simplify deployment of the feature in some ways.